src/data.js

```diff
@@ -50,6 +50,17 @@ module.exports = [
     computer: 'windows',
     phone: 'iphone',
     tags: ['NextJS', 'ReactJS', 'Tailwind CSS', 'FrontEnd'],
+  },
+   {
+    name: 'Tony Blish',
+    description: 'Wannabe Full Stack Developer Photographer',
+    url: 'http://www.studio19and1.com/uses',
+    emoji: '🐌',
+    twitter: '@sonofablish',
+    country: 'US',
```

Comment:
"country" must be one of [🇦🇫, 🇦🇱, 🇩🇿, 🇦🇸, 🇦🇩, 🇦🇴, 🇦🇮, 🇦🇶, 🇦🇬, 🇦🇷, 🇦🇲, 🇦🇼, 🇦🇺, 🇦🇹, 🇦🇿, 🇧🇮, 🇧🇸, 🇧🇭, 🇧🇩, 🇧🇧, 🇧🇾, 🇧🇪, 🇧🇿, 🇧🇯, 🇧🇲, 🇧🇹, 🇧🇴, 🇧🇶, 🇧🇦, 🇧🇼, 🇧🇻, 🇧🇷, 🇧🇳, 🇧🇬, 🇧🇫, 🇰🇭, 🇨🇲, 🇨🇦, 🇨🇻, 🇰🇾, 🇨🇫, 🇹🇩, 🇨🇱, 🇨🇳, 🇨🇽, 🇨🇨, 🇨🇴, 🇰🇲, 🇨🇩, 🇨🇬, 🇨🇰, 🇨🇷, 🇭🇷, 🇨🇺, 🇨🇼, 🇨🇾, 🇨🇿, 🇩🇰, 🇩🇯, 🇩🇲, 🇩🇴, 🇪🇨, 🇪🇬, 🇸🇻, 🇬🇶, 🇪🇷, 🇪🇪, 🇪🇹, 🇫🇴, 🇫🇯, 🇫🇮, 🇫🇷, 🇬🇫, 🇵🇫, 🇹🇫, 🇬🇦, 🇬🇲, 🇬🇪, 🇩🇪, 🇬🇭, 🇬🇮, 🇬🇧, 🇬🇷, 🇬🇱, 🇬🇩, 🇬🇵, 🇬🇺, 🇬🇹, 🇬🇬, 🇬🇳, 🇬🇼, 🇬🇾, 🇭🇹, 🇭🇳, 🇭🇰, 🇭🇺, 🇮🇸, 🇮🇳, 🇮🇩, 🇮🇷, 🇮🇶, 🇮🇪, 🇮🇱, 🇮🇹, 🇨🇮, 🇯🇲, 🇯🇵, 🇯🇪, 🇯🇴, 🇰🇿, 🇰🇪, 🇰🇮, 🇰🇼, 🇰🇬, 🇱🇦, 🇱🇻, 🇱🇧, 🇱🇸, 🇱🇷, 🇱🇾, 🇱🇮, 🇱🇹, 🇱🇺, 🇲🇴, 🇲🇰, 🇲🇬, 🇲🇼, 🇲🇾, 🇲🇻, 🇲🇱, 🇲🇹, 🇲🇭, 🇲🇶, 🇲🇷, 🇲🇺, 🇾🇹, 🇲🇽, 🇫🇲, 🇲🇩, 🇲🇨, 🇲🇳, 🇲🇪, 🇲🇸, 🇲🇦, 🇲🇿, 🇲🇲, 🇳🇦, 🇳🇷, 🇳🇵, 🇳🇱, 🇳🇨, 🇳🇿, 🇳🇮, 🇳🇪, 🇳🇬, 🇳🇺, 🇳🇫, 🇰🇵, 🇲🇵, 🇳🇴, 🇴🇲, 🇵🇰, 🇵🇼, 🇵🇸, 🇵🇦, 🇵🇬, 🇵🇾, 🇵🇪, 🇵🇭, 🇵🇳, 🇵🇱, 🇵🇹, 🇵🇷, 🇶🇦, 🇷🇪, 🇷🇴, 🇷🇺, 🇷🇼, 🇼🇸, 🇸🇲, 🇸🇹, 🇸🇦, 🇸🇳, 🇷🇸, 🇸🇨, 🇸🇱, 🇸🇬, 🇸🇰, 🇸🇮, 🇸🇧, 🇸🇴, 🇿🇦, 🇬🇸, 🇰🇷, 🇸🇸, 🇪🇸, 🇱🇰, 🇸🇩, 🇸🇷, 🇸🇯, 🇸🇿, 🇸🇪, 🇨🇭, 🇸🇾, 🇹🇼, 🇹🇯, 🇹🇿, 🇹🇭, 🇹🇱, 🇹🇬, 🇹🇰, 🇹🇴, 🇹🇹, 🇹🇳, 🇹🇷, 🇹🇲, 🇹🇨, 🇹🇻, 🇺🇬, 🇺🇦, 🇦🇪, 🇺🇳, 🇺🇾, 🇺🇸, 🇺🇿, 🇻🇺, 🇻🇦, 🇻🇪, 🇻🇳, 🇼🇫, 🇪🇭, 🇾🇪, 🇿🇲, 🇿🇼, 🏴󠁧󠁢󠁥󠁮󠁧󠁿, 🏳️‍🌈, 🇪🇺, 🏴󠁧󠁢󠁳󠁣󠁴󠁿]